_posts/2025-08-15-freedom_fuel.md

```diff
@@ -0,0 +1,31 @@
+
+### What fuels the cells
+Combustion, however cozy and heartwarming, is not the only way of extracting energy back out of alcohol (or rather alcohols in general). Fuel cells can convert alcohol into electricity &mdash either directly[^DEFC] or indirectly &mdash; and do it easily and efficiently. They may not outperform hydrogen fuel cells, but since hydrogen is notoriously difficult to store, the lower cost of infrastructure for liquid fuels is a huge advantage. Even more importantly, it is possible to produce alcohols electrochemically[^methanol][^ethanol] &mdash; which opens another possibility of storing energy from renewable sources, further smoothing out the fluctuations in available power.
+
+### Fuel for freedom
+The most unexpected benefit comes from the low-tech nature of ethanol. It is always available &mdash; you just need to mix water and something that contains sugar, let the wild yeast ferment the mash, and then distill the mash using something as simple as a clay retort placed in the middle of a bonfire. Compared to the complexity of refining crude oil, this is trivial &mdash; anyone can do it anywhere. This means that in the event of a zombie apocalypse Brazil will have the highest number of cars still on the road, but it also means that individuals can still make their own fuel in case of an oil shortage, war or global economic collapse. It is impossible to ban, and no corporation can control the whole supply chain tightly enough to force everyone to buy their product. Moreover, the advances in the fuel cell technology and electrochemical synthesis it might actually allow people to efficiently make ethanol with just as much as a single solar panel. That's as close to self-sufficiency as it gets!
+
+Quite ironically, the same substance that harms so many and has been used by oppressors to control the oppressed, might actually pave the way to a greener, more decentralized and more free future. Here's to a brighter future!
```

Comment:
> used by oppressors to control the oppressed

The reference to tsarist practices (if that's what you mean?) is pretty clear in the former Привислинскій край, but it might be less obvious to the international reader &mdash; it might be worth to explain the reference (add a footnote?)

Reply:
Footnote added.